tests/Microsoft.Identity.Test.LabInfrastructure/LabUserHelper.cs

```diff
@@ -172,5 +182,20 @@ public static string FetchUserPassword(string userLabName)
                 throw new InvalidOperationException("Test setup: cannot get the user password. See inner exception.", e);
             }
         }
+
+        public static async Task RevokeUserSessionAsync(LabUser user)
+        {
+            await s_labService.GetLabResponseAsync(Path.Combine(LabApiConstants.LabUserEndPoint, "revokeusersession", user.Upn)).ConfigureAwait(false);
```

Comment:
where these existing lab api options?

why couldn't we have done this directly via Graph API calls? Did it require using some tenant admin account that is not exposed?

Reply:
These APIs are just placeholders and not implemented. Yes, they require tenant admin permissions.